### ISSUE TITLE
error with imports while trying to open a proto file 

### ISSUE THREAD

**Issue by kurtinlane** [troylelandshields/omgrpc#4]
`Error: failed to import '//google/protobuf/timestamp.proto' in `

![screen shot 2017-02-13 at 4 29 51 pm](https://cloud.githubusercontent.com/assets/5891949/22908090/cba8d180-f209-11e6-9fe1-5995d2011464.png)

**Comment by dpratt** [comment]
I get the same error - can you bundle the standard google protobuf extensions, or give us a way to set up a global library/include directory?

**Comment by troylelandshields** [comment]
Yeah that would be good. I'll have to check Node's grpc implementation again and see if there is a way to do this (unless you guys know of one?)

When you generate Go code from a proto that imports from disparate paths it all works fine, but for whatever reason when you try and use the same proto with the node implementation it blows up unless all of the imports and the file itself share some common root directory. It's very annoying. The gRPC package is actively worked on though so maybe it's an issue they've addressed and that I can fix here. I'll check it out.

**Comment by joshadambell** [comment]
Here is another import that fails ` Error: failed to import '/google/api/annotations.proto' `
